builder/build.go

```diff
@@ -909,6 +961,110 @@ func Build(pkgName, outpath string, config *compileopts.Config, action func(Buil
 	})
 }
 
+// createEmbedObjectFile creates a new object file with the given contents, for
+// the embed package.
+func createEmbedObjectFile(data, hexSum, sourceFile, sourceDir, tmpdir string, compilerConfig *compiler.Config) (string, error) {
+	// TODO: this works for small files, but can be a problem for larger files.
```

Comment:
What exactly do you mean "small" here? Please provide specific number.

Reply:
I don't have a specific number and I didn't measure it. But I imagine it might start to slow down compilation when reaching in the megabytes and cause real problems in the gigabytes.